app.js

```diff
@@ -1,19 +1,23 @@
-console.log('js is working');
+console.log('js is linked');
 
 $(document).ready(function() {
-    console.log("Let's get coding!")
+  // indentation
+  console.log("document is loaded")
    
-
   $.ajax({
   	url: "https://api.coindesk.com/v1/bpi/currentprice.json",
   	method: "GET",
   	success: function(currentData) {
   		console.log("success currentdata");
-  		// console.log(currentData);
```

Comment:
Get rid of `console.log`s and commented out code (all "dead code" because they are not necessary for the program to run successfully).